packages/drupal/custom/src/Contact.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Drupal\custom;
+
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\graphql_directives\DirectiveArguments;
+use Drupal\webform\WebformSubmissionInterface;
+
+/**
+ * Helper service to create a contact
+ */
+class Contact {
+  use StringTranslationTrait;
+
+  /**
+   * The custom webform service.
+   *
+   * @var \Drupal\custom\Webform
+   */
+  protected Webform $webformService;
+
+  /**
+   * Contact constructor.
+   */
+  public function __construct(
+    Webform $webformService
+  ) {
+    $this->webformService = $webformService;
+  }
+
+  /**
+   * Creates a contact webform entry.
+   */
+  public function create(DirectiveArguments $args): array {
+    try {
+      $submissionData = [
+        'name' => $args->args['contact']['name'],
```

Comment:
Can we generalize this service to take in arbitrary data? Storing input in webform submissions is a very common case, and then we would not need a separate service and directive for each and could move that into the shared Drupal module!

Reply:
this should be fixed now. The Contact class does not exist anymore and the Webform class is able to create submissions to any webform. The data to be submitted is json encoded.